docs/getting-started/data-collection.md

```diff
@@ -1,4 +1,4 @@
-# Disclaimer: Telemetry
+# Disclaimer: Data Collection
 
 Since you're probably reading this when getting familiar with `zwave-js`, we want to use this opportunity to disclose that `zwave-js` is collecting some data to ensure an optimal experience going forward. The following sections explain which information we collect and why.
```

Comment:
I originally had this under **Getting started**, because that's what a new dev is likely to read. After moving it, this first sentence doesn't make 100% sense. Also, the folder should be renamed to match the section headline better.